go.mod

```diff
@@ -16,6 +16,7 @@ require (
 	gopkg.in/square/go-jose.v2 v2.4.1 // indirect
 	gopkg.in/yaml.v2 v2.2.8
 	helm.sh/helm/v3 v3.2.1
+	honnef.co/go/tools v0.0.0-20190523083050-ea95bdfd59fc // indirect
```

Comment:
Why do we need this pkg ?

Reply:
At this point I do not recall, that was done automagically and it has been to long.  I will investigate.

Reply:
Did some investigation, and the `honnef.co/go/tools` is not used by other repos that use `klog`. In fact the [honnef.co/go/tools](https://github.com/dominikh/go-tools) is is a collection of tools and libraries for working with Go code, including linters and static analysis, most prominently staticcheck.
Would suggest to remove and re-run `go mod tidy && go mod vendor`.